python/python/benchmarks/test_search.py

```diff
@@ -31,16 +31,34 @@ def find_or_clean(dataset_path: Path) -> Union[lance.LanceDataset, None]:
     return None
 
 
+GENRES = [
+    "action",
+    "comedy",
+    "drama",
+    "horror",
+    "romance",
+    "thriller",
+    "western",
+    "sci-fi",
+    "fantasy",
+    "documentary",
+]
+
+
 def create_table(num_rows, offset) -> pa.Table:
     values = pc.random(num_rows * N_DIMS).cast(pa.float32())
     vectors = pa.FixedSizeListArray.from_arrays(values, N_DIMS)
     filterable = pa.array(range(offset, offset + num_rows))
     categories = pa.array(np.random.randint(0, 100, num_rows))
+    genres_values = pa.array(np.random.choice(GENRES, num_rows * 3))
```

Comment:
Could we also test empty list?

Reply:
Empty list is tested in the rust test (it uses the random datagen utilities and I verified they generate empty lists)